### DISCUSSION TITLE
NE and NNE in German PoS tag set. Explanation? (Or documentation error?)

### DISCUSSION THREAD

**Discussion by Gopipo** [explosion/spaCy#9112] [Help: Other Questions] [answered]
As the [Label Scheme documentation of the German pipeline](https://spacy.io/models/de) tells us, the Tagger may use NE and NNE. Is this a documentation error or is there something i'm not seeing? 

.explain() tells us:
>spacy.explain("NNE")
'proper noun'
> spacy.explain("NE")
'proper noun'

I've tested a few sentences and have not found a case where NNE was used. (Thus, it seemingly does adhere to [STTS](https://www.linguistik.hu-berlin.de/de/institut/professuren/korpuslinguistik/mitarbeiter-innen/hagen/STTS_Tagset_Tiger), where NNE does not exist.)

For now, i just check for both to avoid any nasty surprises down the road. (Or are they both translated to PROPN (or similar) behind the scenes?)

**Comment by polm** [comment, marked as answer]
I'm not super familiar with this but it looks like in the Negra corpus NNE was used for a combination of a common and proper noun and that was carried over to Tiger, which we use. I can't find a textual example but I guess it's like "House" in "House of Usher", or maybe some kind of compound with proper and common parts?

You can see it at the [bottom right of A.2 on page 6 here](http://ufal.mff.cuni.cz/~zabokrtsky/other_texts/negra2tgts.pdf), or in [this owl file about STTS](http://acoli.cs.uni-frankfurt.de/resources/olia/stts.owl). I couldn't find it in the public Tiger docs for some reason (though that may be a pdf-related issue).
